src/app/ui/overrides/_adf-info-drawer.scss

```diff
@@ -21,3 +21,30 @@ $icon-size: 48px;
         }
     }
 }
+
+.adf-version-list-container,
+.adf-version-manager-dialog .adf-version-list-container {
+    .adf-version-list {
+        height: auto;
+    }
+
+    .mat-list .mat-list-item {
+        &.mat-3-line {
+            display: flex;
+            align-items: center;
+            height: 100%;
+            min-height: 88px;
```

Comment:
magic number? :)

Reply:
yep :) it is what Material uses by default for Three-line list - as I found on this page https://material.io/guidelines/components/lists.html#lists-specs